netmiko/scp_handler.py

```diff
@@ -106,6 +130,21 @@ def check_file_exists(self, remote_cmd=""):
         else:
             raise ValueError("Unexpected output from check_file_exists")
 
+    def remote_file_size(self, remote_cmd="", remote_file=None, delay_factor=8):
```

Comment:
I set the `delay_factor` to eight here because this is the standard for the functions in `master`.